ListApp/Tests/ListApp.Api.Tests/Extensions/EqualConstraintExtensions.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Collections.Generic;
-using ListApp.Api.Models;
 using NUnit.Framework.Constraints;
+using ListApp.Contracts.Models;
 
 namespace ListApp.Api.Tests.Extensions
 {
```

Comment:
Petr really doesn't like abbreviations, so I'd suggest the whole "constraint". As a bonus, it'll stop lighting up the spellchecker. What I'd suggest as well if you don't have one yet.

Reply:
Thanks for the suggestion, it has never really occurred to me that there might be a spellchecker extension for VS. D'oh. Removed the abbr. in #5ba514c.